docs/heroku.md

```diff
@@ -0,0 +1,23 @@
+# Deploying to Heroku
+
+In order to deploy to heroku, you'll need to run this command once to set a custom
+buildpack:
+
+```
+heroku config:add BUILDPACK_URL=https://github.com/ddollar/heroku-buildpack-multi.git
```

Comment:
Maybe we should switch to use their official support of multi buildpack https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app